package.xml

```diff
@@ -78,8 +78,8 @@
         <required>
             <php>
                 <min>5.4.45</min>
-                <max>5.6.24</max>
+                <max>5.6.26</max>
```

Comment:
I'm running my machine on 5.6.26. So I increased that, I can fix that if needed.